src/Acsint.jl

```diff
@@ -443,7 +430,8 @@ end
         α + β(erf ⍵ R)
         --------------
               R       """
-    PC = P - C
+    PC = Vector{eltype(P)}(undef,3)     # Preallocation needed for Duals 
```

Comment:
What is wrong with `P -C` won't it create a new array of the same type of `P` and `C`?

We might be missing `.` (i.e. `PC = P .- C`) ?

Reply:
I think `P-C` does work. I believe that was from some issues that I had in the past where that vector was being typed as `Vector{Real}` when there were duals in it, but I think that might have gotten fixed with all the work that was done on the type declarations. As for the `.-`, it should have the same behavior for these as just `-` since these are vectors.